### ISSUE TITLE
sealer build -b lite, failed to find image by name: scratch

### ISSUE THREAD

**Issue by yj-cloud** [alibaba/sealer#621]
#### What happened:
```
# sealer build -b lite -t my-kubernetes:test .
...
2021-09-17 16:34:47 [INFO] [lite_builder.go:79] read cluster file /tmp/Clusterfile success !
2021-09-17 16:34:47 [EROR] [build.go:57] failed to find image by name: scratch
# cat Kubefile
FROM scratch
COPY . .
#RUN wget https://docs.projectcalico.org/manifests/tigera-operator.yaml
CMD kubectl apply -f etc/tigera-operator.yaml
CMD kubectl apply -f etc/custom-resources.yaml
# ls
Kubefile  Metadata  README.md  bin  cri  docker  etc  images  kubeadm-config.yaml  registry  scripts  statics
```

#### Environment:
- sealer version (use `sealer version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by kakaZhou719** [comment]
i will fix this issue

**Comment by kakaZhou719** [comment]
hi ,@yj-cloud ,which sealer version you used? seems like this problem not happened in my side.

**Comment by yj-cloud** [comment]
> hi ,@yj-cloud ,which sealer version you used? seems like this problem not happened in my side.

0.4.0, do you use `sealer build -b lite` ?

**Comment by kakaZhou719** [comment]
@yj-cloud ,we fixed this bug at version v0.5.0 .you can use the latest version. any questions pls let me know.

**Comment by bxy4543** [comment]
Version v0.5.0 fixes this problem.
